### DISCUSSION TITLE
Frameworks podrían afectar el rendimiento al instalarlos?

### DISCUSSION THREAD

**Discussion by LuisFerVR** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#492] [Q&A]
El instalar en nuestra lap frameworks podría afectar el rendimiento? o solo un poco cuando se corran, o no afecta para nada? o solo es recomendable instalarlos una vez que estamos seguro de su uso porque si podría afectar?

**Comment by doguedogue** [comment]
No afecta, de hecho tu los arrancas e instalas las cosas a demanda.

```
ng create
npm install
ng serve -o
npm start
etc.
```

**Reply by LuisFerVR** [reply]
Gracias!
